ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/tenant_evpn_vxlan/leaf-tenant-router-bgp-evpn-vlan-aware-bundles.j2

```diff
@@ -5,10 +5,10 @@
 {%         if tenants[tenant].vrfs is defined %}
 {%             for vrf in tenants[tenant].vrfs | arista.avd.natural_sort if leaf.vrfs is not none and vrf in leaf.vrfs %}
     {{ vrf }}:
-      rd: "{{ overlay_loopback_network_summary | ipaddr('network') | ipmath(leaf.id + spine.nodes | length) }}:{{ tenants[tenant].vrfs[vrf].vrf_vni }}"
+      rd: "{{ leaf.evpn_rd_admin_subfield }}:{{ tenants[tenant].vrfs[vrf].vrf_vni }}"
       route_targets:
         both:
-          - "{{ tenants[tenant].vrfs[vrf].vrf_vni }}:{{ tenants[tenant].vrfs[vrf].vrf_vni }}"
+          - "{{ leaf.evpn_rt_admin_subfield or tenants[tenant].vrfs[vrf].vrf_vni }}:{{ tenants[tenant].vrfs[vrf].vrf_vni }}"
```

Comment:
Could we just set a default value in the j2 logic for `leaf.evpn_rt_admin_subfield` with a `else` to catch all unsupported keys (or instantiate with default value)?

Reply:
+1 let's set the default value in the root template `evpn-fabric-l3leaf-yml.j2` and avoid the `or` operator.

Reply:
Well everything is possible, but remember that here we have a default value of "vni" so a value that will be different per service. So I chose to set the default value of "false" and then let the final value be resolved in the service templates. Alternative would be to copy over the same loops and vni logic to the l3leaf template and store an extra rd_admin_subfield value per service.

Reply:
Another alternative could be to leave "vni" as the default value and add an if/else in the service templates:
if leaf.evpn_rt_admin_subfield != "vni"
    rd : <leaf.evpn_rt_admin_subfield>:<vni>
else
    rd : <vni>:<vni>
endif

Reply:
@ClausHolbechArista, I like your idea and you are absolutely right about the vni value.

I like the first proposal: 

```{% if leaf.evpn_rt_admin_subfield == "vni" %}
    rd : {{vni}}:{{vni}}
{% else %}
    rd : {{leaf.evpn_rt_admin_subfield}}:{{vni}}
{% endif %}```

IMO it is the most readable!

Once adjustments are made, I will merge!

Reply:
As discussed we will settle on inline "or" for now.